tests/telescope_extension_spec.lua

```diff
@@ -0,0 +1,37 @@
+local helpers = require('tests.support.helpers')
+
+describe('telescope extension', function()
```

Comment:
can you find a way to use telescope itself to test the picker? ideally install telescope locally using a helper (maybe cloning the repo).